source/examples/ServerManualCodeExamples.kt

```diff
@@ -120,43 +123,571 @@ fun main() = runBlocking {
     // End Example 6
 
     // Start Example 9
-    val flowFindEquality = collection.withDocumentClass<Document>()
+    val findFlow = collection.withDocumentClass<Document>()
```

Comment:
Q: Given collection already is a `MongoCollection<Document>` why do the examples all use the `withDocumentClass`?

Reply:
I'll remove!